internal/provider/models.go

```diff
@@ -86,6 +86,7 @@ type Dashboard struct {
 	Group         string               `json:"group,omitempty"`
```

Comment:
We don't want to advertise Group (it was an old thing left over from Beacon days)